jsapp/js/components/textBox.es6

```diff
@@ -0,0 +1,131 @@
+/**
```

Comment:
I think this file should be deleted, as there is one already in `js/components/common` :eyes: